apps/event-manager-program/programs/event-manager/src/collections/event.rs

```diff
@@ -33,6 +33,7 @@ pub struct Event {
   pub attendance_mint_bump: u8,             // 1
   pub temporal_vault_bump: u8,              // 1
   pub gain_vault_bump: u8,                  // 1
+  pub f_id: String,                         // (40 + 4)
```

Comment:
I wouldn't store the Firebase ID in the account, instead, I'd use the Firebase ID for the Event's PDA.